libraries/ui/src/Section.tsx

```diff
@@ -1,55 +1,57 @@
 import React from 'react';
 import clsx from 'clsx';
-import { CTALinkOrButton } from './CTALinkOrButton';
 
-export type SectionProps = React.PropsWithChildren<{
-  className?: string,
-  title?: string,
-  titleLevel?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6',
-  subtitle?: string | React.ReactNode,
-  ctaText?: string,
-  ctaUrl?: string,
-}>;
+type BaseProps = {
+  className?: string;
+  title?: React.ReactNode;
+  titleLevel?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+  subtitle?: React.ReactNode;
+  subtitleLevel?: 'p' | 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+  rightNode?: React.ReactNode;
+};
 
-export const Section: React.FC<SectionProps> = ({
-  className, title, titleLevel = 'h2', subtitle, ctaText, ctaUrl, children,
+export type SectionHeadingProps = BaseProps;
+export type SectionProps = React.PropsWithChildren<BaseProps>;
+
+export const SectionHeading: React.FC<BaseProps> = ({
```

Comment:
should: Section probably shouldn't have all the title / subtitle props, given these could be composable.